ci-operator/config/openshift/hypershift/openshift-hypershift-main__kubevirt.yaml

```diff
@@ -109,6 +109,44 @@ tests:
       resources:
         requests:
           cpu: 100m
+    - as: kv-e2e-external-infra
+      cli: latest
+      commands: |-
+        # install hypershift
+        bin/hypershift install --hypershift-image ${HYPERSHIFT_RELEASE_LATEST}
+
+        # compile the e2e tests
+        make e2e
```

Comment:
fyi, I don't think you have to run these commands again. It doesn't hurt anything, it's just a no-op

Reply:
I agree.
Removed redundant commands.